src/Evolution/Initialization/ConservativeSystem.hpp

```diff
@@ -101,6 +101,9 @@ struct ConservativeSystem {
       const Parallel::ConstGlobalCache<Metavariables>& /*cache*/,
       const ArrayIndex& /*array_index*/, ActionList /*meta*/,
       const ParallelComponent* const /*meta*/) noexcept {
+    ERROR(
+        "Could not find dependency 'Initialization::Tags::InitialTime' in "
+        "DataBox.");
     return {std::move(box)};
```

Comment:
You can remove this line. I don't know what's the best-practice for this.

Reply:
It should be removed, I think :+1: